gapis/service/service.proto

```diff
@@ -420,6 +421,21 @@ message MemoryRange {
   uint64 size = 2;
 }
 
+// UsageHints hints to the server the intended usage of the result of a request.
+// This can be used to improve performance and responsiveness of the RPCs.
+message UsageHints {
+ // Preview indicates that the request has been made for a thumbnail or
+ // low-quality representation of the underlying data. Previews are considered
+ // non-critical and non-urgent; the server may consider scheduling other work
+ // ahead of previews, and possibly delay the processing of the request to
+ // batch together requests.
+ bool preview = 1;
+
+ // Primary indicates that the request has been made for the primary view.
+ // Primary requests are prioritized and are low-latency.
+ bool primary = 2;
```

Comment:
I am not quite sure what "primary" means.... Any other name suggestions?

focus? - i.e. data required for something the user clicked-on